cmd/crio/config.go

```diff
@@ -166,6 +166,14 @@ default_mounts = [
 # Maximum number of processes allowed in a container.
 pids_limit = {{ .PidsLimit }}
 
+# Run an init inside containers that forwards signals and reaps processes
```

Comment:
Can we condense these to just one setting: `init`? If empty string, then it does nothing, else use the binary.

Reply:
yeah I like that better